### ISSUE TITLE
In_kafka & In_kafka_group not working dur to a bug in new ruby-kafka release

### ISSUE THREAD

**Issue by AJouaneau-CS** [fluent/fluent-plugin-kafka#280]
Since 15 July 2019 ruby-kafka-0.7.9 is out with a [bug on custom loggers](https://github.com/zendesk/ruby-kafka/issues/756). So In_kafka & In_kafka_group are crashing almost instantly 

```
 2019-07-16 16:00:19 +0000 [info]: #0 starting fluentd worker pid=136 ppid=1 worker=0
2019-07-16 16:00:19 +0000 [error]: #0 unexpected error error_class=TypeError error="no implicit conversion of Fluent::Log into String"
  2019-07-16 16:00:19 +0000 [error]: #0 /usr/lib/ruby/2.3.0/logger.rb:703:in `open'
  2019-07-16 16:00:19 +0000 [error]: #0 /usr/lib/ruby/2.3.0/logger.rb:703:in `open_logfile'
  2019-07-16 16:00:19 +0000 [error]: #0 /usr/lib/ruby/2.3.0/logger.rb:695:in `set_dev'
  2019-07-16 16:00:19 +0000 [error]: #0 /usr/lib/ruby/2.3.0/logger.rb:635:in `initialize'
  2019-07-16 16:00:19 +0000 [error]: #0 /usr/lib/ruby/2.3.0/logger.rb:353:in `new'
  2019-07-16 16:00:19 +0000 [error]: #0 /usr/lib/ruby/2.3.0/logger.rb:353:in `initialize'
  2019-07-16 16:00:19 +0000 [error]: #0 /var/lib/gems/2.3.0/gems/ruby-kafka-0.7.9/lib/kafka/tagged_logger.rb:63:in `new'
  2019-07-16 16:00:19 +0000 [error]: #0 /var/lib/gems/2.3.0/gems/ruby-kafka-0.7.9/lib/kafka/tagged_logger.rb:63:in `initialize'
  2019-07-16 16:00:19 +0000 [error]: #0 /var/lib/gems/2.3.0/gems/ruby-kafka-0.7.9/lib/kafka/tagged_logger.rb:56:in `new'
  2019-07-16 16:00:19 +0000 [error]: #0 /var/lib/gems/2.3.0/gems/ruby-kafka-0.7.9/lib/kafka/tagged_logger.rb:56:in `new'
  2019-07-16 16:00:19 +0000 [error]: #0 /var/lib/gems/2.3.0/gems/ruby-kafka-0.7.9/lib/kafka/client.rb:75:in `initialize'
  2019-07-16 16:00:19 +0000 [error]: #0 /var/lib/gems/2.3.0/gems/ruby-kafka-0.7.9/lib/kafka.rb:366:in `new'
  2019-07-16 16:00:19 +0000 [error]: #0 /var/lib/gems/2.3.0/gems/ruby-kafka-0.7.9/lib/kafka.rb:366:in `new'
  2019-07-16 16:00:19 +0000 [error]: #0 /var/lib/gems/2.3.0/gems/fluent-plugin-kafka-0.10.0/lib/fluent/plugin/in_kafka.rb:188:in `start'
  2019-07-16 16:00:19 +0000 [error]: #0 /var/lib/gems/2.3.0/gems/fluentd-1.5.1/lib/fluent/compat/call_super_mixin.rb:42:in `start'
  2019-07-16 16:00:19 +0000 [error]: #0 /var/lib/gems/2.3.0/gems/fluentd-1.5.1/lib/fluent/root_agent.rb:203:in `block in start'
  2019-07-16 16:00:19 +0000 [error]: #0 /var/lib/gems/2.3.0/gems/fluentd-1.5.1/lib/fluent/root_agent.rb:192:in `block (2 levels) in lifecycle'
  2019-07-16 16:00:19 +0000 [error]: #0 /var/lib/gems/2.3.0/gems/fluentd-1.5.1/lib/fluent/root_agent.rb:191:in `each'
  2019-07-16 16:00:19 +0000 [error]: #0 /var/lib/gems/2.3.0/gems/fluentd-1.5.1/lib/fluent/root_agent.rb:191:in `block in lifecycle'
  2019-07-16 16:00:19 +0000 [error]: #0 /var/lib/gems/2.3.0/gems/fluentd-1.5.1/lib/fluent/root_agent.rb:178:in `each'
  2019-07-16 16:00:19 +0000 [error]: #0 /var/lib/gems/2.3.0/gems/fluentd-1.5.1/lib/fluent/root_agent.rb:178:in `lifecycle'
  2019-07-16 16:00:19 +0000 [error]: #0 /var/lib/gems/2.3.0/gems/fluentd-1.5.1/lib/fluent/root_agent.rb:202:in `start'
  2019-07-16 16:00:19 +0000 [error]: #0 /var/lib/gems/2.3.0/gems/fluentd-1.5.1/lib/fluent/engine.rb:274:in `start'
  2019-07-16 16:00:19 +0000 [error]: #0 /var/lib/gems/2.3.0/gems/fluentd-1.5.1/lib/fluent/engine.rb:219:in `run'
  2019-07-16 16:00:19 +0000 [error]: #0 /var/lib/gems/2.3.0/gems/fluentd-1.5.1/lib/fluent/supervisor.rb:805:in `run_engine'
  2019-07-16 16:00:19 +0000 [error]: #0 /var/lib/gems/2.3.0/gems/fluentd-1.5.1/lib/fluent/supervisor.rb:549:in `block in run_worker'
  2019-07-16 16:00:19 +0000 [error]: #0 /var/lib/gems/2.3.0/gems/fluentd-1.5.1/lib/fluent/supervisor.rb:730:in `main_process'
  2019-07-16 16:00:19 +0000 [error]: #0 /var/lib/gems/2.3.0/gems/fluentd-1.5.1/lib/fluent/supervisor.rb:544:in `run_worker'
  2019-07-16 16:00:19 +0000 [error]: #0 /var/lib/gems/2.3.0/gems/fluentd-1.5.1/lib/fluent/command/fluentd.rb:316:in `<top (required)>'
  2019-07-16 16:00:19 +0000 [error]: #0 /usr/lib/ruby/2.3.0/rubygems/core_ext/kernel_require.rb:55:in `require'
  2019-07-16 16:00:19 +0000 [error]: #0 /usr/lib/ruby/2.3.0/rubygems/core_ext/kernel_require.rb:55:in `require'
  2019-07-16 16:00:19 +0000 [error]: #0 /var/lib/gems/2.3.0/gems/fluentd-1.5.1/bin/fluentd:8:in `<top (required)>'
  2019-07-16 16:00:19 +0000 [error]: #0 /usr/local/bin/fluentd:22:in `load'
  2019-07-16 16:00:19 +0000 [error]: #0 /usr/local/bin/fluentd:22:in `<main>'
```

**Comment by repeatedly** [comment]
I just released v0.11.0 with this patch: https://github.com/fluent/fluent-plugin-kafka/pull/281
Try this parameter to avoid ruby-kafka's bug.

**Comment by AJouaneau-CS** [comment]
Works nice, thank you..
